creating-virtual-machines/virtualized-hardware-configuration.adoc

```diff
@@ -161,6 +161,53 @@ spec:
 You can check list of available models
 https://github.com/libvirt/libvirt/blob/master/src/cpu_map/index.xml[here].
 
+Enabling CPUNodeDiscovery feature-gate
```

Comment:
The gate name is not really that important, what about calling it Cpu compatibility enforcement or something?